syntax = "proto3";

package pose_detector;

import "pose.proto";

message DetectPoseRequest {
  string frame_id = 1;
}

message DetectPoseResponse {
  pose.PoseReading pose = 1;
}
